pkg/cmd/admin/node/listpods.go

```diff
@@ -38,6 +38,17 @@ func (l *ListPodsOptions) Run() error {
 		}
 	}
 
+	// determine if printer kind is json or yaml and modify output
+	// to combine all pod lists into a single list
+	switch printer.(type) {
+	case *kubectl.JSONPrinter:
```

Comment:
type checking these smells bad... why not flatten the list for all custom output types?